lib/modules/datasource/glasskube-packages/schema.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from 'zod';
+import { Yaml } from '../../../util/schema-utils';
+
+export const GlasskubePackageVersions = z.object({
```

Comment:
```suggestion
const GlasskubePackageVersions = z.object({
```
we don't use it directly